packages/wallet/src/redux/types.ts

```diff
@@ -1,4 +1,10 @@
-export const enum PlayerIndex {
+export const enum TwoPartyPlayerIndex {
   A = 0,
   B = 1,
 }
+
+export const enum ThreePartyPlayerIndex {
```

Comment:
Do we foresee any cases with 4 parties coming up? Should we start moving away from two and three party cases toward n-party structures? Or is the added complexity not worth it?

Reply:
I don't know if we'll ever get rid of all of the reliance on two parties. If we do, then we should get rid of the `*PlayerIndex` types, for sure.